src/pages/survey/answer/[surveyId]/index.tsx

```diff
@@ -114,13 +114,16 @@ function SurveyResultsPage() {
             createDate={createDate}
           />
 
-          <div className="mt-10 mb-4 flex justify-center">
-            <Toggle
-              isEnabled={showOnlyWithExtraFeedback}
-              onToggle={toggleShowOnlyWithExtraFeedback}
-              label="With extra feedback only"
-            />
-          </div>
+          {filteredAnswersData.length >= 1 && (
```

Comment:
`items` is also bad idea because if we will have only answers without extra feedback and checked checkbox we will not able to unswitch it. We should use length of all answers without filtering.

Reply:
Thanks for the advice guys! I just return one more variable(answersData) in the surveyResultsManager.ts so I can access in index.jsx as the all answers data